src/GameLogic/PlayerActions/ItemConsumeActions/ItemUpgradeConsumeHandlerPlugIn.cs

```diff
@@ -34,14 +35,9 @@ public enum ItemFailResult
         None,
 
         /// <summary>
-        /// Sets the option to level one.
+        /// Sets the option to the base level.
         /// </summary>
-        SetOptionToLevelOne,
-
-        /// <summary>
-        /// Decreases the option by one level.
-        /// </summary>
-        DecreaseOptionByOne,
```

Comment:
I think we still need DecreaseOptionByOne for Jewel of Life

Reply:
Sorry, I have completely overlooked the unit tests :sweat_smile: 
I will have a look.

Reply:
https://github.com/MUnique/OpenMU/pull/537